front/public/locales/en/timesStops.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "Name",
+  "stop_time": "Stop Time",
```

According to deepL :)
```suggestion
  "stop_time": "Stopping Time",
```

---

Done